libs/libfoundation/include/libfoundation/AssetManager.h

```diff
@@ -0,0 +1,20 @@
+// Begining of possible AssetManager to locate/read/write to files and folders in the app directory
+namespace LFoundation {
+
+class AssetManager {
+public:
+    AssetManager(std::string name) {
```

Comment:
Use reference passing a std::string: like `const std::string& name`.

Also you might consider implementing a constructor like
```
AssetManager(const std::string& name)
     : m_app_root("/Applications/" + name + ".app/Content/")
{
}
```